Tiny.RestClient/TinyRestClient.cs

```diff
@@ -274,12 +282,17 @@ public IRequest DeleteRequest(string route = null)
             Request tinyRequest,
             CancellationToken cancellationToken)
         {
-            using (var content = await CreateContentAsync(tinyRequest.Content, cancellationToken).ConfigureAwait(false))
+            await ExecuteDynamicHeadersAsync(tinyRequest);
```

Comment:
ExecuteDynamicHeadersAsync must be executed on SendRequestAsync function just before to add headers